stl/inc/__msvc_int128.hpp

```diff
@@ -1367,6 +1382,28 @@ struct _Signed128 : _Base128 {
         _Word[1] |= _That._Word[1];
         return *this;
     }
+
+    template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
+    _NODISCARD constexpr explicit operator _Ty() const noexcept {
+        bool _Negative = int64_t(_Word[1]) < 0;
+        const _Unsigned128 _Abs{_Word[0], abs(int64_t(_Word[1]))};

```

Comment:
It's wrong to use `abs` here - it triggers UB for the most negative value and is not constexpr-compatible.

Reply:
Oh sorry I still have the habits from when I wrote the elc... 😂
fixed at [`a711aa9` (#3559)](https://github.com/microsoft/STL/pull/3559/commits/a711aa9fde1b4e64b44c6eb6c54dd204f4c9fa49)